xclim/sdba/measures.py

```diff
@@ -496,6 +500,17 @@ def _taylordiagram(
         }
     )
 
+    # Normalize the standard deviations byt the standard deviation of the reference.
+    if normalize:
+        if (out[{"taylor_param": 0}] == 0).any():
+            raise ValueError(
+                "`ref_std =0` (homogeneous field) obtained, normalization is not possible."
+            )
+        out[{"taylor_param": [0, 1]}] = (
+            out[{"taylor_param": [0, 1]}] / out[{"taylor_param": 0}]
+        )
+        out.attrs["normalized"] = True
```

Comment:
```suggestion
        out.attrs["normalized"] = True
```
Assign this attribute if there is normalization, and don't add anything otherwise, is it reasonable?

Reply:
I think this makes sense. This is solely for the user to consult manually, right?

Reply:
Yes, it's not used elsewhere, thanks